proto/zitadel/feature/v2/instance.proto

```diff
@@ -11,6 +11,8 @@ import "zitadel/feature/v2/feature.proto";
 option go_package = "github.com/zitadel/zitadel/pkg/grpc/feature/v2;feature";
 
 message SetInstanceFeaturesRequest{
+  reserved 6;
+  reserved "actions";
```

Comment:
Our buf linting settings are currently on FILE, which prevents removing fields (even with reserved).
Let me quickly clarify how we want to handle this.

Reply:
i left it on FILE, but added `FIELD_NO_DELETE_UNLESS_NAME_RESERVED` and `FIELD_NO_DELETE_UNLESS_NUMBER_RESERVED` and made an exection for `FIELD_NO_DELETE`

i'll add a note in the API design